{{cookiecutter.project_slug}}/gulpfile.js

```diff
@@ -67,72 +74,80 @@ gulp.task('styles', function() {
       ]
     }).on('error', sass.logError))
     .pipe(plumber()) // Checks for errors
-    .pipe(autoprefixer({browsers: ['last 2 versions']})) // Adds vendor prefixes
-    .pipe(pixrem())  // add fallbacks for rem units
-    .pipe(gulp.dest(paths.css))
+    .pipe(postcss(processCss))
+    .pipe(dest(paths.css))
     .pipe(rename({ suffix: '.min' }))
-    .pipe(cssnano()) // Minifies the result
-    .pipe(gulp.dest(paths.css));
-});
+    .pipe(postcss(minifyCss)) // Minifies the result
+    .pipe(dest(paths.css))
+}
 
 // Javascript minification
-gulp.task('scripts', function() {
-  return gulp.src(paths.js + '/project.js')
+function scripts() {
+  return src(paths.js + '/project.js')
     .pipe(plumber()) // Checks for errors
     .pipe(uglify()) // Minifies the js
     .pipe(rename({ suffix: '.min' }))
-    .pipe(gulp.dest(paths.js));
-});
-
+    .pipe(dest(paths.js))
+}
 
 {% if cookiecutter.custom_bootstrap_compilation == 'y' %}
 // Vendor Javascript minification
-gulp.task('vendor-scripts', function() {
-  return gulp.src(paths.vendorsJs)
+function vendorScripts() {
+  return src(paths.vendorsJs)
     .pipe(concat('vendors.js'))
-    .pipe(gulp.dest(paths.js))
+    .pipe(dest(paths.js))
     .pipe(plumber()) // Checks for errors
     .pipe(uglify()) // Minifies the js
     .pipe(rename({ suffix: '.min' }))
-    .pipe(gulp.dest(paths.js));
-});
+    .pipe(dest(paths.js))
+}
 {% endif %}
 
 // Image compression
-gulp.task('imgCompression', function(){
-  return gulp.src(paths.images + '/*')
+function imgCompression() {
+  return src(paths.images + '/*')
     .pipe(imagemin()) // Compresses PNG, JPEG, GIF and SVG images
-    .pipe(gulp.dest(paths.images))
-});
+    .pipe(dest(paths.images))
+}
+
+// Generate all assets
+const generateAssets = parallel(
```

Comment:
I noticed that this task was added. By looking deeper it's replacing run-sequence, which has in its readme:

> This was intended to be a temporary solution until the release of gulp 4.0 which should have support for defining task dependencies similarly.

That looks like a nice addition to Gulp 4.0 👍 

I'm thinking that this task could be defined further down, just above the `dev` one: they are both "meta" tasks running other tasks in parallel.

Reply:
Yep I'm happy to do that; it makes sense to me either way around.